moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/compiler/AnnotationsProcessor.java

```diff
@@ -3841,7 +3841,7 @@ private JavaClass processXmlElementDecl(JavaClass type, JavaMethod next, Package
                 XmlElementDecl elementDecl = (XmlElementDecl) helper.getAnnotation(next, XmlElementDecl.class);
                 url = elementDecl.namespace();
                 localName = elementDecl.name();
-                scopeClass = elementDecl.scope();
+                scopeClass = (Class<XmlElementDecl.GLOBAL>) elementDecl.scope();
```

Comment:
`Global` is default but one of many possibly values, see https://jakarta.ee/specifications/xml-binding/3.0/apidocs/jakarta.xml.bind/jakarta/xml/bind/annotation/xmlelementdecl#scope() for details

Reply:
Changed to Class<?>